stdlib/dis.pyi

```diff
@@ -70,6 +70,9 @@ else:
         starts_line: int | None
         is_jump_target: bool
 
+class Instruction(_Instruction):
+    def _disassemble(self, lineno_width: int = ..., mark_as_current: bool = ..., offset_width: int = ...) -> str: ...
```

Comment:
```suggestion
class Instruction(_Instruction):
    def _disassemble(self, lineno_width: int = 3, mark_as_current: bool = False, offset_width: int = 4) -> str: ...
```

Reply:
Thanks, I wasn't sure about preferred style for that.